### PR TITLE
Rename 'supervision' plugin to 'monitoring'

### LINKED ISSUE #58: Rename the "Supervision" section into "Monitoring"
Even though the word "Supervision" exists in English, it sounds more like a French expression. Seems to me the standard name for this in the industry is "Monitoring" or "Overview" if you want to separate Monitoring 2 parts  :  "Alerts" + "Overview"

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #58

### CODE REVIEW

**Comment by bersace** [comment]
I suggest to duplicate `[supervision]` sections as `[monitoring]`in agent docker images to ease upgrade of temboard. This can be done now and must be cleaned once images are up to date.

https://github.com/dalibo/docker/blob/master/temboard/temboard-agent/9.6/temboard-agent.conf#L58-L72

**Comment by pgiraud** [comment]
This pull request is now ready for review.
Please note that it only takes the UI into account. Compatibility with agents is maintained by continuing to collect data using `supervision` routes.
Most of the work was related to Flake8.

File: rpm/temboardui.plugins.monitoring.model.orm.py.patch [hunk 1]
```diff
@@ -1,5 +1,5 @@
---- temboard-ui-0.0.1/temboardui/plugins/supervision/model/orm.py.orig	2016-07-15 21:34:18.031001950 +0200
```
Comment (bersace) [inline rpm/temboardui.plugins.monitoring.model.orm.py.patch]: What is this patch ?

Comment (pgiraud) [inline rpm/temboardui.plugins.monitoring.model.orm.py.patch]: I have no idea. See https://github.com/dalibo/temboard/blob/master/rpm/temboardui.plugins.supervision.model.orm.py.patch
Do you think that it should be left untouched?

Comment (bersace) [inline rpm/temboardui.plugins.monitoring.model.orm.py.patch]: oh, ok, it's for rpm. Let's fix this once we restore packaging. No problem.

File: doc/temboard-repository-setup.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The `repository` is a PostgreSQL (>=9.5) database. It let temboard store its data, including users, agent registration and metrics data.
 
-Usage of plugin `supervision` requires `tablefunc` extension. This extension is available as part of the extensions shipped with the source code of PostgreSQL, in the `contrib/` directory. Usually, the "contrib" package of PostgreSQL from your Linux distribution has it.
+Usage of plugin `monitoring` requires `tablefunc` extension. This extension is available as part of the extensions shipped with the source code of PostgreSQL, in the `contrib/` directory. Usually, the "contrib" package of PostgreSQL from your Linux distribution has it.
```
Comment (julmon) [inline doc/temboard-repository-setup.md]: This line can be removed, `tablefunc` extension is not required anymore.

File: doc/temboard-repository-setup.md [hunk 3]
```diff
@@ -40,10 +40,10 @@ If the PostgreSQL cluster is on th different host, replace `127.0.0.1/32` by the
 psql -U temboard -1 -v'ON_ERROR_STOP=on' -f /usr/share/temboard/application.sql temboard
 ```
 
-If you plan to use the plugin `supervision`:
+If you plan to use the plugin `monitoring`:
 ```
 sudo -u postgres psql -U postgres -c "CREATE EXTENSION tablefunc" temboard
```
Comment (julmon) [inline doc/temboard-repository-setup.md]: This line can be removed too.

**Pull request merged**